Drivers/cmake/arm-none-eabi.cmake

```diff
@@ -0,0 +1,46 @@
+SET(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR arm)
+include(CMakeForceCompiler)
+set(TOOLCHAIN arm-none-eabi)
+if(NOT DEFINED TOOLCHAIN_PREFIX)
+    if(CMAKE_HOST_SYSTEM_NAME STREQUAL Linux)
+        set(TOOLCHAIN_PREFIX "/usr")
+    elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+        set(TOOLCHAIN_PREFIX "/usr/local")
+    elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+        message(STATUS "Please specify the TOOLCHAIN_PREFIX !\n For example: -DTOOLCHAIN_PREFIX=\"C:/Program Files/GNU Tools ARM Embedded\" ")
+    else()
+        set(TOOLCHAIN_PREFIX "/usr")
+        message(STATUS "No TOOLCHAIN_PREFIX specified, using default: " ${TOOLCHAIN_PREFIX})
+    endif()
+endif()
+set(TOOLCHAIN_BIN_DIR ${TOOLCHAIN_PREFIX}/bin)
+set(TOOLCHAIN_INC_DIR ${TOOLCHAIN_PREFIX}/${TOOLCHAIN}/include)
```

Comment:
Where is this used?

Reply:
Indeed, this doesn't seem to be used, I took it out now.